tests/Unit/Application/User/Command/ConvertAccessRequestToUserCommandHandlerTest.php

```diff
@@ -0,0 +1,342 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Unit\Application\User\Command;
+
+use App\Application\IdFactoryInterface;
+use App\Application\User\Command\ConvertAccessRequestToUserCommand;
+use App\Application\User\Command\ConvertAccessRequestToUserCommandHandler;
+use App\Domain\User\AccessRequest;
+use App\Domain\User\Exception\AccessRequestNotFoundException;
+use App\Domain\User\Exception\SiretMissingException;
+use App\Domain\User\Exception\UserAlreadyRegisteredException;
+use App\Domain\User\Organization;
+use App\Domain\User\Repository\AccessRequestRepositoryInterface;
+use App\Domain\User\Repository\OrganizationRepositoryInterface;
+use App\Domain\User\Repository\UserRepositoryInterface;
+use App\Domain\User\User;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+final class ConvertAccessRequestToUserCommandHandlerTest extends TestCase
+{
+    private MockObject $idFactory;
+    private MockObject $accessRequestRepository;
+    private MockObject $userRepository;
+    private MockObject $organizationRepository;
+    private MockObject $accessRequest;
+
+    public function setUp(): void
+    {
+        $this->idFactory = $this->createMock(IdFactoryInterface::class);
+        $this->accessRequestRepository = $this->createMock(AccessRequestRepositoryInterface::class);
+        $this->userRepository = $this->createMock(UserRepositoryInterface::class);
+        $this->organizationRepository = $this->createMock(OrganizationRepositoryInterface::class);
+        $this->accessRequest = $this->createMock(AccessRequest::class);
+    }
+
+    public function testConvertWithOrganizationThanksToSiret(): void
```

Comment:
le nom de la fonction m'a surpris le" thanks to"  mais peut être que c'est clair pour d'autres personnes

Reply:
J'avoue ne pas être trop inspiré, qu'est-ce que tu mettrais à la place ?